frontend/src/components/Card.tsx

```diff
@@ -94,14 +107,31 @@ const Card = ({
           </div>
         ) : null}
       </div>
+
       {/* Link to project name if provided */}
       {projectName && (
         <Link href={projectLink} rel="noopener noreferrer" className="mt-2 font-medium">
           {projectName}
         </Link>
       )}
+
       {/* Render project summary using Markdown */}
       <Markdown content={summary} className="py-2 pr-4 text-gray-600 dark:text-gray-300" />
+
```

Comment:
I can see you pass in these to modal as well, but I don't see anything on the modal. 
![Screenshot 2025-03-27 at 6 21 16 PM](https://github.com/user-attachments/assets/fbbd5f53-cc07-4c98-bd0b-860e67bd38e5)
![Screenshot 2025-03-27 at 6 21 21 PM](https://github.com/user-attachments/assets/be8e0ec7-809e-428b-b425-5ee46404f168)

And I'm not sure if it's useful to have them on the modal. What was your idea behind it?